### PR TITLE
Remove workarounds for old OpenVINO

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: tests/python/accuracy/public_scope.json [hunk 1]
```diff
@@ -100,24 +100,24 @@
...
-                        "(535, 374, 638, 479, 0.535, 1, RBC)",
-                        "(513, 8, 633, 151, 0.454, 1, RBC)",
-                        "(197, 86, 412, 286, 0.405, 1, RBC)",
-                        "(21, 291, 143, 399, 0.401, 1, RBC)"
-                    ]
-                }
-            ]
-        }
+        ]
+    },
```
Comment (AlexKoff88) [inline tests/python/accuracy/public_scope.json:104]: Any idea what caused the difference?

**Comment by AlexKoff88** [comment]
Before merging this we need to make sure that Geti and OTX moved to the new OpenVINO version. Usually, it is fixed AFAIK.
@Radwan-Ibrahim, @sovrasov can you please comment here?

**Comment by sovrasov** [comment]
Thin is planned to happen this sprint. I'm working on upgrade of OTX, then Geti will be upgraded as well.

File: tests/python/accuracy/public_scope.json (hunk 1 above)
Comment (Wovchena) [reply tests/python/accuracy/public_scope.json:104]: There was a nested test entry marked as TODO. Now I moved it out of one layer of nesting to outside to allow running it

**Comment by Wovchena** [comment]
> Thin is planned to happen this sprint. I'm working on upgrade of OTX, then Geti will be upgraded as well.

This is also aligned what we discussed couple of weeks ago about on a sync meeting.

So, I need someone's approve, to merge this :)

**Review by AlexKoff88 — APPROVED** [review]

**Pull request merged**